hid-ghlive/src/hid-ghlive.c

```diff
@@ -220,8 +222,23 @@ static int ghlive_probe(struct hid_device *hdev,
 		timer_setup(&sc->poke_timer, ghl_magic_poke_ps3wiiu, 0);
 	}
 	else if (sc->quirks & GHL_GUITAR_PS4){
+		
+		struct usb_interface *intf = to_usb_interface(sc->hdev->dev.parent);
+
+		if (intf->cur_altsetting->desc.bNumEndpoints != 2)
+		return -ENODEV;
+				
+		for (i = 0; i < 2; i++) {
```

Comment:
I based myself on the xpad.c code here for this FOR loop. But, I am not convinced that i should use "2". I think maybe I could use the intf->cur_altsetting->desc.bNumEndpoints used on line 228 which was verified to be = 2. What do you think?

Reply:
Please make the update, yes.